known-shortcuts/pocket.ts

```diff
@@ -1,3 +1,5 @@
+import { AppParams } from "~/types";
+
 const app = {
```

Comment:
:zap: **Suggestion** 
The `app` object is being cast to `AppParams` using the `as` keyword. While this is a valid TypeScript pattern, it is generally safer to use type annotations directly in the object definition to ensure that the object conforms to the `AppParams` type at compile time.